modules/installation-initializing.adoc

```diff
@@ -265,16 +277,17 @@ installation program.
 endif::aws[]
 ifdef::azure[]
 ... Select *azure* as the platform to target.
-... If you do not have a Microsoft Azure profile stored on your computer, specify the
-following Azure parameter values for your subscription and service principal:
-**** *azure subscription id*: The subscription ID to use for the cluster.
-Specify the `id` value in your account output.
-**** *azure tenant id*: The tenant ID. Specify the `tenantId` value in your
-account output.
-**** *azure service principal client id*: The value of the `appId` parameter
-for the service principal.
-**** *azure service principal client secret*: The value of the `password`
-parameter for the service principal.
+... If you do have a Microsoft Azure profile stored on the computer, specify the following Azure parameter values for your subscription:
+**** *azure subscription id*: Enter the subscription ID to use for the cluster.
+**** *azure tenant id*: Enter the tenant ID.
+... Depending on the Azure identity you are using to deploy the cluster, do one of the following when prompted for the *azure service principal client id*:
+**** If you are using a service principal, enter its application ID.
+**** If you are using a system-assigned managed identity, leave this value blank.
+**** If you are using a user-assigned managed identity, specify its client ID.
+... Depending on the Azure identity you are using to deploy the cluster, do one of the following when prompted for the *azure service principal client secret*:
+**** If you are using a service principal, enter its password.
+**** If you are using a system-assigned managed identity, leave this value blank.
+**** If you are using a user-assigned managed identity,leave this value blank.
```

Comment:
Upon a closer look, I am not as concerned about the user experience as I was when I first reached out to you @r4f4 , but let's take a look at this to make sure I have it correct.